src/PushNotifications.Api/Controllers/SubscriptionsTracker/GetTopicCounterController.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Discovery.Contracts;
+using Elders.Web.Api;
+using PushNotifications.Api.Attributes;
+using PushNotifications.Api.Converters;
+using PushNotifications.Contracts;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Web.Http;
+using System.Web.Http.ModelBinding;
+
+namespace PushNotifications.Api.Controllers.SubscriptionsTracker
+{
+    [RoutePrefix("TopicCounter")]
```

Comment:
I would design the routs in a way if they were external.

Reply:
I don't really understand what this means